### ISSUE TITLE
Explanation on test set (Novel and similar split)

### ISSUE THREAD

**Issue by zubair-irshad** [graspnet/graspnet-baseline#122]
Hi authors, 

Great work on the Graspnet-1B dataset and thank you for releasing it. Quick question: Are your test set i.e. test_novel and test_similar completely different categories or different instances within the same categories as seen in the training set? Appreciate your help!

**Comment by Fang-Haoshu** [comment]
Hi, there is no category in our dataset, just instance
